input/intro-notes/StructureDefinition-au-coverage-intro.md

```diff
@@ -2,3 +2,11 @@
 
 **Profile specific implementation guidance:**
 - See each Identifier profile page for guidance related to that identifier type.
+- When populating the type using the preferred ValueSet :
+    - Medicare coverage **SHOULD** be represented by sending the V3 Value SetActCoverageTypeCode code "PUBLICPOL"
```

Comment:
The PR description specifices that terms like Medicare, DVA, Workcover etc should be italised. Please update the PR to be consistent with the description.

Reply:
Amended, thanks.

Reply:
Confirmed